03/app.js

```diff
@@ -11,4 +11,27 @@ const buttonSettings = {
         color: '#444'
     },
     text: 'Click me!',
-}
\ No newline at end of file
+}
+
+const btnParent = document.querySelectorAll('.parent-for-button');
+
+if (btnParent) {
+    const newButton = document.createElement('button');
+
+    newButton.setAttribute("class", 'btn');
+    newButton.setAttribute("title", 'super button');
+
+    newButton.style.color = '#444';
+    newButton.style.border = '1px solid #336699';
+    newButton.style.padding = '5px 20px'
```

Comment:
Może spróbuj wykorzystać do tego pętlę np. `for ... in` oraz dane ze zmiennej `buttonSettings`